operations/helm/charts/mimir-distributed/enable-zoneawareness.yaml

```diff
@@ -0,0 +1,171 @@
+rollout_operator:
+  enabled: true
+
+store_gateway:
+  zone_aware_replication:
```

Comment:
No replicas set?

Reply:
Well, if it is not set, then the default 1 will be rounded up to the number of zones, so you'll get 1 each